corehq/apps/es/tests/test_esquery.py

```diff
@@ -204,3 +205,32 @@ def test_sort(self):
             {"timeStart": {"order": "asc"}},
         ]
         self.checkQuery(query.sort('timeStart', reset_sort=False), json_output)
+
+    def test_cleanup_before_run(self):
```

Comment:
Yeah. agreed it's not the best test, but I balked at writing up more test code than necessary since it would probably only get used right here.